packages/notifier/src/publish-kit.js

```diff
@@ -102,6 +111,19 @@ export const subscribeLatest = subscriber => {
 };
 harden(subscribeLatest);
 
+export const PublisherI = M.interface('Publisher', {
+  publish: M.call(M.any()).returns(),
+  finish: M.call(M.any()).returns(),
+  fail: M.call(M.any()).returns(),
+});
+export const SubscriberI = M.interface('Subscriber', {
+  subscribeAfter: M.call().optional(M.bigint()).returns(M.promise()),
+});
+export const publishKitIKit = harden({
```

Comment:
```suggestion
export const publishIKit = harden({
```
or perhaps just
```suggestion
export const publishKit = harden({
```

Reply:
I think this name needs to have a form like "publishKit\<NonemptySuffix>", and chose "IKit" as the suffix to match prevailing patterns. But the resulting name _is_ awkward... what about "publishKitShape"?

Reply:
Agree it needs a suffix. `IKit` is the prevailing suffix. Another option, matching the param name, would be `publishKitGuardKit`. 
https://github.com/Agoric/agoric-sdk/blob/96979ceb43e7c04e6f93707eb198405859a797d4/packages/vat-data/src/far-class-utils.js#L167

Another option is to simply inline it. Why is it exported? What else might have to verify an argument matches this shape?

Reply:
I use PublisherI and SubscriberI in Zoe.  They're delivered as a pair, so I'd like to use the canonical declaration.

Reply:
Got it. Since the current pattern is typename + `IKit` for the interface guard of that type, I propose it stay this way. Coming up with a different scheme is outside the scope of this PR.